data/mods/Magiclysm/monsters/Ogre.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "id": "mon_ogre",
+    "type": "MONSTER",
+    "name": { "str": "Ogre" },
+    "description": "Green beast known for living in swamps and eating humans, it will kill anything it gets its hands on.",
+    "default_faction": "ogre",
+    "bodytype": "human",
+    "species": [ "OGRE" ],
+    "volume": "100 L",
+    "weight": "130 kg",
+    "hp": 140,
+    "speed": 123,
+    "material": [ "flesh" ],
+    "symbol": "O",
+    "color": "green",
+    "aggression": 75,
+    "morale": 100,
+    "melee_skill": 7,
+    "melee_dice": 5,
+    "melee_dice_sides": 8,
+    "dodge": 5,
+    "harvest": "ogre",
+    "anger_triggers": [ "HURT", "FIRE", "PLAYER_WEAK" ],
```

Comment:
Fire as an anger trigger? Interesting; what is the logic?

Reply:
I'm going to make Ogres fire resistant in order to make them harder to fight, also it makes sense for them to not be harmed by fire as ogres are sometimes depicted as being immune to fire.

Reply:
> ogres are sometimes depicted as being immune to fire

Oh? Interesting. I was thinking it was related to their living in swamps.

Reply:
> > ogres are sometimes depicted as being immune to fire
> 
> Oh? I didn't recall that. I was thinking it was related to their living in swamps.

It could also be the swamp thing, as they would be wet if always near swamps. Some examples of Ogres that are immune to or hurt less by fire: Fire Ogres( Orcs Must Die), Type of Ogre immune to fire in that game, Shrek( Set on fire by a dragon during the movie, feels minimal pain), Firebellies( Warhammer), tribe of ogres that worship a volcano god, breath fire and are immune to fire-based attacks.